lib/simplifyExpression/arithmetic/index.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
```

Comment:
hmm do you think 'arithmetic' is specific enough? or would people be confused about what it did (my guess it that'd be okay, but part of me thinks maybe not so I wanted to bring it up)